server/cmd/server/start.go

```diff
@@ -67,6 +69,12 @@ func (s *server) Start(ctx context.Context) {
 	log := logging.FromContext(ctx)
 	router := gin.New()
 	router.Use(gin.LoggerWithConfig(gin.LoggerConfig{SkipPaths: []string{"/livez"}}))
+	router.Use(cors.New(cors.Config{
+		AllowOrigins:     []string{"https://localhost:8443", "https://localhost:8444"}, // s.options.CorsAllowedOrigins,
```

Comment:
I want to change the value of `AllowOrigins` to `s.options.CorsAllowedOrigins,` but that will lead to all the test failing on CircleCi https://github.com/numaproj/numaflow/actions/runs/8529227991/job/23364549970 but i am not sure why. I debugged and it fails on this line not finding the pods https://github.com/numaproj/numaflow/blob/main/Makefile#L231

Reply:
What do you see when you test it on you local?